Pod/Classes/ResizingTextView.swift

```diff
@@ -14,6 +14,16 @@ open class ResizingTextView: UITextView {
     
     private var heightConstraint: NSLayoutConstraint!
     
+    private var maxHeight: CGFloat?
+    
+    public var resizeForNumberOfLines: Int = 0 {
+        didSet {
+            guard resizeForNumberOfLines > 0 else { return (maxHeight = nil) }
+            let padding = (textContainer.lineFragmentPadding * 2) + textContainerInset.top + textContainerInset.bottom
+            maxHeight = (font!.lineHeight * CGFloat(resizeForNumberOfLines)) + padding
```

Comment:
We shouldn't be explicitly unwrapping `font!`. Looks like it can be nil. https://stackoverflow.com/questions/19471467/uitextview-font-is-nil

If the `maxHeight` is being used and the font is `nil`, we should fail to handle the height restriction and just log an error to tell the developer that this is wrong. "Max height cannot be calculated when font is 'nil' for \(self)".